src/service/apiClient.js

```diff
@@ -71,12 +71,51 @@ async function request(method, endpoint, data, auth = true) {
   return response.json();
 }
 
+async function deletePost(postId) {
+  console.log(postId)
+  const token = localStorage.getItem('token'); 
+  const response = await fetch(`${API_URL}/posts/${postId}`, {
```

Comment:
better to make a `delete` function that can be used by other functions rather than writing the fetch in here. Take a look at the posts/gets we have